infra/ohdsi_webapi.bicep

```diff
@@ -30,114 +29,157 @@ resource webApp 'Microsoft.Web/sites@2022-03-01' = {
-      }
-      {
-        name: 'DATASOURCE_OHDSI_SCHEMA'
-        value: postgresWebApiSchemaName
-      }
-      {
-        name: 'DATASOURCE_USERNAME'
-        value: postgresWebapiAppUsername
-      }
-      {
-        name: 'DATASOURCE_PASSWORD'
-        value: '@Microsoft.KeyVault(VaultName=${keyVaultName};SecretName=${postgresWebapiAppSecret})'
-      }
-      {
-        name: 'DATASOURCE_URL'
-        value: '@Microsoft.KeyVault(VaultName=${keyVaultName};SecretName=${jdbcConnectionStringWebapiAdminSecret})'
-      }
-      {
-        name: 'FLYWAY_BASELINEDESCRIPTION'
-        value: 'Base Migration'
-      }
-      {
-        name: 'FLYWAY_BASELINEONMIGRATE'
-        value: 'true'
-      }
-      {
-        name: 'flyway_baselineVersionAsString'
-        value: flywayBaselineVersion
-      }
-      {
-        name: 'FLYWAY_DATASOURCE_DRIVERCLASSNAME'
-        value: 'org.postgresql.Driver'
-      }
-      {
-        name: 'FLYWAY_DATASOURCE_USERNAME'
-        value: postgresWebapiAdminUsername
-      }
-      {
-       name: 'FLYWAY_DATASOURCE_PASSWORD'
-       value: '@Microsoft.KeyVault(VaultName=${keyVaultName};SecretName=${postgresWebapiAdminSecret})'
-      }
-      {
-        name: 'FLYWAY_DATASOURCE_URL'
-        value: '@Microsoft.KeyVault(VaultName=${keyVaultName};SecretName=${jdbcConnectionStringWebapiAdminSecret})'
-      }
-      {
-        name: 'FLYWAY_LOCATIONS'
-        value: 'classpath:db/migration/postgresql'
-      }
-      {
-        name: 'FLYWAY_PLACEHOLDERS_OHDSISCHEMA'
-        value: postgresWebApiSchemaName
-      }
-      {
-        name: 'FLYWAY_SCHEMAS'
-        value: postgresWebApiSchemaName
-      }
-      {
-        name: 'FLYWAY_TABLE'
-        value: 'schema_history'
-      }
-      {
-        name: 'SECURITY_SSL_ENABLED'
-        value: 'false'
-      }
-      // TODO: Add security app settings
-      {
-        name: 'SPRING_BATCH_REPOSITORY_TABLEPREFIX'
-        value: 'webapi.BATCH_'
-      }
-      {
-        name: 'SPRING_JPA_PROPERTIES_HIBERNATE_DEFAULT_SCHEMA'
-        value: postgresWebApiSchemaName
-      }
-      {
-        name: 'SPRING_JPA_PROPERTIES_HIBERNATE_DIALECT'
-        value: 'org.hibernate.dialect.PostgreSQLDialect'
-      }
-      {
-        name: 'WEBSITE_HEALTHCHECK_MAXPINGFAILURES'
-        value: '10'
-      }
-      {
-        name: 'WEBSITE_HTTPLOGGING_RETENTION_DAYS'
-        value: '30'
-      }
-      {
-        name: 'WEBSITES_CONTAINER_START_TIME_LIMIT'
-        value: '1800'
-      }
-      {
-        name: 'WEBSITES_ENABLE_APP_SERVICE_STORAGE'
-        value: 'false'
-      }
-      {
-        name: 'WEBSITES_PORT'
-        value: '8080'
-      }
-
-    ] 
+        {
+          name: 'DOCKER_REGISTRY_SERVER_URL'
+          value: dockerRegistryServer
+        }
+        {
+          name: 'DATASOURCE_DRIVERCLASSNAME'
+          value: 'org.postgresql.Driver'
+        }
+        {
+          name: 'DATASOURCE_OHDSI_SCHEMA'
+          value: postgresWebApiSchemaName
```

Comment:
Do you want to make it as hardcoded 'webapi' as we discussed or should it be done in a separate PR?

Reply:
I didn't change it yet as I'm not 100% sure and it can wait.